app/views/helpers/TimelineEvents.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.helpers
+
+import java.time.ZonedDateTime
+
+import models.declaration.notifications.Notification
+import play.api.i18n.Messages
+import play.twirl.api.Html
+
+case class TimelineEvent(title: String, dateTime: ZonedDateTime, dateTimeAsShown: String, content: Option[Html]) extends Ordered[TimelineEvent] {
```

Comment:
Nit: Do we want two versions of this event's dateTime (a real date and a string version of it)? Seems a violation of the 'single source of truth' rule.

Reply:
Point is that `Timeline` is a reusable generic partial. In a different page we could want a different format of `dateTime`. In any case, the `dateTime` field in `TimelineEvent` is not important at all (I would say even optional). It's only used for the html attribute `datetime` of tag `time` and it does not matter even if it is different from `dateTimeAsShown`. It was my choice to add it to the tag for completeness, but it is of no use for us.

Reply:
I can see `dateTime` field is used to populate `datetime` attribute in the html. Although, I am not sure if you really need this field. If you're saying it is not really used, then I'd remove that from the case class. I am aware that if you don't remove it now, we'll get stuck with it forever.

Reply:
Actually, it was a nice suggestion because it forced me to find a better solution :slightly_smiling_face: